### PR TITLE
[sleepiq] Add thing type update instructions

### PR DESCRIPTION
**Pull request by mhilbush**
Add thing type instructions for updating thing with new channels.

This is a follow up to PR #14447, which added 4 new channels.

Signed-off-by: Mark Hilbush [mark@hilbush.com](mailto:mark@hilbush.com)

### CODE REVIEW

**Comment by mhilbush** [comment]
@J-N-K Can you please have a look?

File: bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 		<properties>
 			<property name="vendor">Select Comfort</property>
+			<property name="thingTypeVersion">1</property>
```
Comment (J-N-K) [inline bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/thing/thing-types.xml:14]: This should be in the updated thing, not in the bridge.

File: bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/update/instructions.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="sleepiq:dualBed">
+
+		<instruction-set targetVersion="1">
+			<add-channel id="todaySleepInBedSeconds">
```
Comment (J-N-K) [inline bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/update/instructions.xml:9]: If I see understand the binding correctly, the channel shall be available for both channel groups, correct? I need to check if that is supported or if I need to add something more.

File: bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/thing/thing-types.xml (hunk 1 above)
Comment (mhilbush) [reply bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/thing/thing-types.xml]: Yes, of course. Fixed.

File: bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/update/instructions.xml (hunk 2 above)
Comment (mhilbush) [reply bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/update/instructions.xml:9]: Yes, that's correct.

Comment (J-N-K) [reply bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/update/instructions.xml:9]: https://github.com/openhab/openhab-core/pull/3391 will bring the solution for this.

Comment (J-N-K) [inline bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/update/instructions.xml:9]:
```suggestion
			<add-channel id="todaySleepInBedSeconds" groupIds="left,right">
```

Comment (J-N-K) [reply bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/update/instructions.xml:9]: Also below

**Review by J-N-K — APPROVED** [review]
LGTM

**Comment by mhilbush** [comment]
Now that openhab/openhab-core#3391 is merged, this should be ready for merge.

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**